skipchain/struct.go

```diff
@@ -373,7 +374,7 @@ func (sb *SkipBlock) Copy() *SkipBlock {
 	b := &SkipBlock{
 		SkipBlockFix: sb.SkipBlockFix.Copy(),
 		Hash:         make([]byte, len(sb.Hash)),
-		ForwardLink:  make([]*BlockLink, len(sb.ForwardLink)),
+		ForwardLink:  make([]*ForwardLink, len(sb.ForwardLink)),
```

Comment:
With this change, you have invalidated all the existing .DB files. We will need to re-establish the production network. However, we also need to change the rosters in the production conodes to refer to each other via TLS. So we need to do both at the same time.

We need a better strategy for evolving on-disk formats. Is now the time to do it? Or maybe with the first change after v2?

Reply:
> We need a better strategy for evolving on-disk formats. Is now the time to do it? 

it's definitely a good time to think about it

Reply:
+1 - I opened #1059 for this discussion. For the moment we're in 'API-breaking mode', so the things will break regularly anyway. But for the changes in the upcoming `v2` we definitively must have a good way of defining that.